geomesa-blobstore/README.md

```diff
@@ -1,3 +1,116 @@
 # GeoMesa Blob Store
 
 See [GeoMesa Blob Store](../docs/user/blobstore.rst) in the [GeoMesa User Manual](http://geomesa.org/documentation).
+=======
+#GeoMesa BlobStore
```

Comment:
Is it a good idea to have the same text both here and in ``docs/user/blobstore.rst``? The idea of having the "See this section in the user manual" above was to avoid having to keep two separate texts in sync (or worse, not keeping them in sync).